modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -1,16 +1,16 @@
 = Create and Configure Endpoints
 
-You can create endpoints in the host or partner page, or from the Message Flow configuration wizard. 
+You can create endpoints in the host or partner page, or from the Message Flow configuration wizard.
 
 == Creating Endpoints From the Partner Page
 
-Create and configure new send or receive endpoints in the partner page: 
+Create and configure new send or receive endpoints in the partner page:
 
 . In the left navigation menu, select *Partners*.
-. Select the partner for which to create and configure the endpoint.  
+. Select the partner for which to create and configure the endpoint.
 . Click *New* in the *Endpoints: Receive from <partner-name>* or *Endpoints: Send to <partner-name>* section.
-. <<configure-endpoint,Configure the endpoint>> according to the type of endpoint you are creating and the protocol you selected. 
-. Click *Save*. 
+. <<configure-endpoint,Configure the endpoint>> according to the type of endpoint you are creating and the protocol you selected.
```

Comment:
I think this format is correct. The inline link in this step goes to a  page that lists the different endpoint types since there are so many. It would not make sense to maintain that in the topic.

Reply:
Lines 20-24 appear for me - I will verify this after I finish.